studybuddies/src/app/services/login/login.service.ts

```diff
@@ -0,0 +1,49 @@
+import { Injectable, EventEmitter } from '@angular/core';
+import { User } from '../../models/user';
+import { HttpClient } from "@angular/common/http";
+import 'rxjs';
+import { HttpHeaders } from "@angular/common/http";
+
+@Injectable({
+  providedIn: 'root'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7202914)
```suggestion
  providedIn: "root"
```